com.discordapp.Discord.appdata.xml

```diff
@@ -20,6 +20,8 @@
       <li>Available on Linux, macOS, Windows, iOS, Android, and your web browser.</li>
     </ul>
     <p>Discord is written in Javascript, React, Elixir, and Rust. The desktop application uses the electron framework.</p>
+    <p>The flatpak version runs in a sandbox to provide better safety and privacy for users. However, this sandboxing prevents the following features from working out of the box: Game Activity, Drag and Drop Files, Rich Presence. Check the README in the Github repo for details.</p>
```

Comment:
You can put a real link here for the readme.

Reply:
Is it HMTL, like <a href... ?

Reply:
Reading the spec it doesn't actually support URLs, sorry for the misunderstanding.